client/reader/sidebar/index.jsx

```diff
@@ -342,13 +344,17 @@ export default withCurrentRoute(
 				sectionGroup,
 				sectionName
 			);
+			const postKey = pickBy( { feedId: +feedId, blogId: +blogId, postId: +postId } );
+			const post = getPostByKey( state, postKey ) || { _state: 'pending' };
```

Comment:
We simply need this to be able to pass the post object to the onClose callback for the tracking event that uses this object. Unfortunately, trying to get the post from state in the controller itself (where we define the close callback) is too early to retrieve the post so we use it as a callback instead.